projects/ssr-tests/src/utils/log.utils.ts

```diff
@@ -27,25 +27,64 @@ export function clearSsrLogFile(): void {
 
 /**
  * Validates that all lines starting with `{` are valid JSON objects.
- * Otherwise it throws an error.
+ * Otherwise it warns.
  *
  * Note: multi-line JSONs (printed by SSR in dev mode) cannot be parsed by `JSON.parse`.
  *       That's why we need to to run SSR in prod mode to get single line JSON logs.
  */
-function validateJsonsInLogs(logs: string[]): void {
-  logs.forEach((text) => {
-    if (text.charAt(0) === '{') {
+function validateJsonsInLogs(rawLogs: string[]): void {
```

Comment:
`medium:` Can you provide steps to verify the warning locally? Btw. the SSR testing framework is getting more complex and maybe we should consider covering by testing at least the most crucial parts of utils

Reply:
Good question.
After rethinking, I believe we shouldn't validate logs in runtime of tests to detect if the app is built in dev mode. But instead we should check if the app is built in prod mode, before running the tests.
here created a PR for it: https://github.com/SAP/spartacus/pull/19334

And in this PR I'm leaving only the un-skip of the test